tests/test_grid.py

```diff
@@ -69,21 +73,6 @@ def test_concatenate(self):
                                        [3, 4, 5, 6, 7], [4, 5, 6, 7, 8]])
         np.testing.assert_array_equal(fd1.axes_labels, fd.axes_labels)
 
-    def test_concatenate(self):
```

Comment:
Is there a specific reason to eliminate this test?

Reply:
It was a duplicate.